manifests/plugin/sensors.pp

```diff
@@ -1,21 +1,27 @@
 # https://collectd.org/wiki/index.php/Plugin:Sensors
 class collectd::plugin::sensors (
-  $ensure           = 'present',
-  $manage_package   = undef,
-  $sensorconfigfile = undef,
-  $sensor           = undef,
-  $ignoreselected   = undef,
-  $interval         = undef,
+  $ensure                  = 'present',
+  $manage_package          = undef,
+  $sensorconfigfile        = undef,
+  $sensor                  = undef,
+  $ignoreselected          = undef,
+  $interval                = undef,
+  $package_install_options = $collectd::package_install_options,
 ) {
 
   include ::collectd
 
   $_manage_package = pick($manage_package, $::collectd::manage_package)
 
+  if $package_install_options != undef {
+    validate_array($package_install_options)
```

Comment:
I copied the code from the apache.pp file, for all occurances. Is there any plan on you guys refactoring the entire code ? Actually none of the RedHat occurences uses the correct options.
Will see what and how I can fix this.